entc/gen/template/meta.tmpl

```diff
@@ -48,9 +48,10 @@ const (
 {{/* Has at least one field (not enum) with default value */}}
 {{ $fields := $.Fields }}{{ if $.HasOneFieldID }}{{ if $.ID.UserDefined }}{{ $fields = append $fields $.ID }}{{ end }}{{ end }}
 {{ $hasDefault := false }}{{ range $f := $fields }}{{ if and $f.Default (not $f.IsEnum) }}{{ $hasDefault = true }}{{ end }}{{ end }}
+{{ $hasUpdateDefault := false }}{{ range $f := $fields }}{{ if and $f.UpdateDefault (not $f.IsEnum) }}{{ $hasUpdateDefault = true }}{{ end }}{{ end }}
```

Comment:
This can be improved, we could use only a single loop I guess.

I only duplicated because I'm not sure I understood why we avoid `enum` fields.

Reply:
Enum default values are handled close to their declaration and do not support `UpdateDefault`. 
```suggestion
```

See my suggestion below.